lib/experimental/diff.nim

```diff
@@ -9,6 +9,28 @@
 
 ## This module implements an algorithm to compute the
 ## `diff`:idx: between two sequences of lines.
+##
+## A basic example of ``diffInt`` on 2 arrays of integers:
+##
+## .. code::nim
+##
+##   import experimental/diff
+##   echo diffInt([0, 1, 2, 3, 4, 5, 6, 7, 8], [-1, 1, 2, 3, 4, 5, 666, 7, 42])
```

Comment:
Examples containing `echo` like this don't show the result of a function, so this doesn't really show what function does (I needed to copy-paste this example to personally see what it does).

See what is done in the documentation of other modules.

Either have a commented line where you write a result of an echo, or use `assert` in your examples.